lib/cocoapods/installer/analyzer/target_inspector.rb

```diff
@@ -95,13 +97,12 @@ def compute_project_path
         #         encountered target is returned (it is assumed to be the one
         #         to integrate in simple projects).
         #
-        # @param  [Xcodeproj::Project] user_project
-        #         the user project
+        # @param  [Array<PBXNativeTarget>] native_targets
+        #         all native targets in the user project
         #
-        # @return [Array<PBXNativeTarget>]
+        # @return [Array<PBXNativeTarget>] the targets to be integrated
         #
-        def compute_targets(user_project)
-          native_targets = user_project.native_targets
+        def compute_targets(native_targets)
```

Comment:
changed to accept just the array of native targets, since that's all it was using the project for